MixedOps-NIReconnect-6k-28m fails on `master` and `release/0.15`

<!--  Thanks for submitting a bug report!  Before submitting:
1. Try searching the existing issues to see if your issue has already been reported
2. If you're reporting a security vulnerability, please email security@hedera.com instead of opening an issue
-->

**Summary of the defect**
`MixedOps-NIReconnect-6k-28m` 
test fails with `HapiGetVersionInfo{}` throwing `DUPLICATE_TRANSACTION` error on branch `master`.
test fails with `HapiGetVersionInfo{}` throwing `INSUFFICIENT_TX_FEE ` error on branch `release/0.15`.

**Service logs (if applicable)**
```
2021-06-27 05:52:39.827 INFO   143  CreateAccountsBeforeReconnect - -------------- STARTING CreateAccountsBeforeReconnect SUITE --------------
2021-06-27 05:52:39.829 INFO   57   MapPropertySource - Initializing a MapPropertySource from {nodes=35.229.47.197:0.0.3:50211,34.139.35.117:0.0.4:50211,35.231.233.40:0.0.5:50211,34.138.75.109:0.0.6:50211,34.138.220.127:0.0.7:50211,34.139.249.227:0.0.8:50211, totalClients=6, default.node=0.0.4, tls=off, txn=ALTERNATE, node.selector=fixed, default.payer=0.0.1002, ci.properties.map=tps=50,mins=2,threads=30,testTopicId=120000}
2021-06-27 05:52:39.866 INFO   143  HapiApiClients - Constructed 96 new stubs building clients for HapiApiClients{node0=NodeConnectInfo{host=35.229.47.197, port=50211, tlsPort=50212, account=0.0.3}, node1=NodeConnectInfo{host=34.139.35.117, port=50211, tlsPort=50212, account=0.0.4}, node2=NodeConnectInfo{host=35.231.233.40, port=50211, tlsPort=50212, account=0.0.5}, node3=NodeConnectInfo{host=34.138.75.109, port=50211, tlsPort=50212, account=0.0.6}, node4=NodeConnectInfo{host=34.138.220.127, port=50211, tlsPort=50212, account=0.0.7}, node5=NodeConnectInfo{host=34.139.249.227, port=50211, tlsPort=50212, account=0.0.8}, default=0.0.4}
2021-06-27 05:52:39.875 INFO   57   MapPropertySource - Initializing a MapPropertySource from {mins=2, tps=50, threads=30, testTopicId=120000}
2021-06-27 05:52:39.908 INFO   138  FeesAndRatesProvider - The exchange rates are :: [Current hbar/cent equiv 1 <-> 12 expiry @ 4102444800, Next hbar/cent equiv 1 <-> 15 expiry @ 4102444800]
2021-06-27 05:52:39.933 INFO   156  FeesAndRatesProvider - The fee schedule covers 56 ops.
2021-06-27 05:52:39.936 INFO   42   CustomSpecAssert - PerfTestLoadSettings{tps=120, totalClients=1, mins=3, tolerance=5, burstSize=5, allowedSecsBelow=60, threads=1, totalTestAccounts=2, totalTestTopics=1, totalTestTokens=1, durationCreateTokenAssociation=60, durationTokenTransfer=60, testActiveTokenAccounts=2, testTreasureStartAccount=1001, submitMessageSize=256, submitMessageSizeVar=64, memoLength=25, totalTokenAssociations=0, totalScheduledTransactions=0, initialBalance=100000000000000, testTopicId=30000, balancesExportPeriodSecs=60, clientToExportBalances=false}
2021-06-27 05:52:39.937 INFO   322  HapiApiSpec - 'RunCreateAccounts' finished initial execution of LogMessage{}
2021-06-27 05:52:39.958 INFO   256  RunLoadTest - Thread thread0 ops 1 current TPS 55.555553
2021-06-27 05:52:49.965 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.03998
2021-06-27 05:52:59.969 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.06398
2021-06-27 05:53:09.972 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.06398
2021-06-27 05:53:19.976 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.06398
2021-06-27 05:53:29.980 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.07597
2021-06-27 05:53:39.984 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.06398
2021-06-27 05:53:49.987 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.07597
2021-06-27 05:53:59.990 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.07597
2021-06-27 05:54:09.994 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.06398
2021-06-27 05:54:19.998 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.06398
2021-06-27 05:54:30.001 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.06398
2021-06-27 05:54:40.005 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.06398
2021-06-27 05:54:50.009 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.06398
2021-06-27 05:55:00.013 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.07597
2021-06-27 05:55:10.017 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.06398
2021-06-27 05:55:20.021 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.05197
2021-06-27 05:55:30.026 INFO   256  RunLoadTest - Thread thread0 ops 1201 current TPS 120.06398
2021-06-27 05:55:32.840 ERROR  38   CustomSpecAssert - Operation 'HapiGetVersionInfo{}' :: Bad answerOnlyPrecheck! expected OK, actual DUPLICATE_TRANSACTION
Exception in thread "thread0" java.lang.IllegalStateException: com.hedera.services.bdd.spec.exceptions.HapiQueryPrecheckStateException: Bad answerOnlyPrecheck! expected OK, actual DUPLICATE_TRANSACTION
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:39)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:45)
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.testRun(RunLoadTest.java:247)
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.lambda$threadMode$22(RunLoadTest.java:215)
	at java.base/java.lang.Thread.run(Thread.java:835)
Caused by: com.hedera.services.bdd.spec.exceptions.HapiQueryPrecheckStateException: Bad answerOnlyPrecheck! expected OK, actual DUPLICATE_TRANSACTION
	at com.hedera.services.bdd.spec.queries.HapiQueryOp.submitOp(HapiQueryOp.java:186)
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:194)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:36)
	... 4 more
2021-06-27 05:55:32.842 INFO   229  RunLoadTest - Total Ops submitted 0, actual TPS 0.0
2021-06-27 05:55:32.842 INFO   322  HapiApiSpec - 'RunCreateAccounts' finished initial execution of RunLoadTest{}
2021-06-27 05:55:33.331 INFO   341  HapiApiSpec - 'RunCreateAccounts' - final status: PASSED!
2021-06-27 05:55:33.333 INFO   174  CreateAccountsBeforeReconnect - -------------- RESULTS OF CreateAccountsBeforeReconnect SUITE --------------
2021-06-27 05:55:33.333 INFO   176  CreateAccountsBeforeReconnect - Spec{name=RunCreateAccounts, status=PASSED}
```
**Additional Context**
[master slack link](https://hedera-hashgraph.slack.com/archives/C018Y4E6ADT/p1624776618112400)
[release/0.15 slack link](https://hedera-hashgraph.slack.com/archives/CKWHL8R9A/p1624777729126600)

--- a/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/CreateAccountsBeforeReconnect.java
+++ b/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/CreateAccountsBeforeReconnect.java
@@ -32,10 +32,10 @@ import java.util.concurrent.atomic.AtomicInteger;
 import java.util.function.Supplier;
 
 import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
-import static com.hedera.services.bdd.spec.queries.QueryVerbs.getVersionInfo;
 import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoCreate;
 import static com.hedera.services.bdd.spec.utilops.LoadTest.defaultLoadTest;
 import static com.hedera.services.bdd.spec.utilops.UtilVerbs.logIt;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.noOp;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.BUSY;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.DUPLICATE_TRANSACTION;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.PLATFORM_TRANSACTION_NOT_CREATED;
@@ -66,10 +66,7 @@ public class CreateAccountsBeforeReconnect extends HapiApiSuite {
 	private synchronized HapiSpecOperation generateCreateAccountOperation() {
 		final long accNumber = accountNumber.getAndIncrement();
 		if (accNumber >= ACCOUNT_CREATION_LIMIT) {
-			return getVersionInfo()
-					.fee(ONE_HUNDRED_HBARS)
-					.payingWith(GENESIS)
-					.noLogging();
+			return noOp();
 		}
 
 		return cryptoCreate("account" + accNumber)

--- a/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/CreateFilesBeforeReconnect.java
+++ b/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/CreateFilesBeforeReconnect.java
@@ -32,10 +32,10 @@ import java.util.concurrent.atomic.AtomicInteger;
 import java.util.function.Supplier;
 
 import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
-import static com.hedera.services.bdd.spec.queries.QueryVerbs.getVersionInfo;
 import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileCreate;
 import static com.hedera.services.bdd.spec.utilops.LoadTest.defaultLoadTest;
 import static com.hedera.services.bdd.spec.utilops.UtilVerbs.logIt;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.noOp;
 import static com.hedera.services.bdd.suites.reconnect.CreateAccountsBeforeReconnect.DEFAULT_MINS_FOR_RECONNECT_TESTS;
 import static com.hedera.services.bdd.suites.reconnect.CreateAccountsBeforeReconnect.DEFAULT_THREADS_FOR_RECONNECT_TESTS;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.BUSY;
@@ -64,10 +64,7 @@ public class CreateFilesBeforeReconnect extends HapiApiSuite {
 	private synchronized HapiSpecOperation generateFileCreateOperation() {
 		final long file = fileNumber.getAndIncrement();
 		if (file >= FILE_CREATION_LIMIT) {
-			return getVersionInfo()
-					.fee(ONE_HUNDRED_HBARS)
-					.payingWith(GENESIS)
-					.noLogging();
+			return noOp();
 		}
 
 		return fileCreate("file" + file)

--- a/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/CreateSchedulesBeforeReconnect.java
+++ b/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/CreateSchedulesBeforeReconnect.java
@@ -35,7 +35,6 @@ import java.util.function.Supplier;
 import static com.cedarsoftware.util.UrlUtilities.getHostName;
 import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
 import static com.hedera.services.bdd.spec.infrastructure.OpProvider.STANDARD_PERMISSIBLE_PRECHECKS;
-import static com.hedera.services.bdd.spec.queries.QueryVerbs.getVersionInfo;
 import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoCreate;
 import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoTransfer;
 import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileUpdate;
@@ -44,6 +43,7 @@ import static com.hedera.services.bdd.spec.transactions.crypto.HapiCryptoTransfe
 import static com.hedera.services.bdd.spec.utilops.LoadTest.defaultLoadTest;
 import static com.hedera.services.bdd.spec.utilops.LoadTest.initialBalance;
 import static com.hedera.services.bdd.spec.utilops.UtilVerbs.logIt;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.noOp;
 import static com.hedera.services.bdd.spec.utilops.UtilVerbs.sleepFor;
 import static com.hedera.services.bdd.suites.perf.PerfUtilOps.scheduleOpsEnablement;
 import static com.hedera.services.bdd.suites.reconnect.CreateAccountsBeforeReconnect.DEFAULT_MINS_FOR_RECONNECT_TESTS;
@@ -73,10 +73,7 @@ public class CreateSchedulesBeforeReconnect extends HapiApiSuite {
 
 	private HapiSpecOperation generateScheduleCreateOperation() {
 		if (scheduleNumber.getAndIncrement() > SCHEDULE_CREATION_LIMIT) {
-			return getVersionInfo()
-					.fee(ONE_HUNDRED_HBARS)
-					.payingWith(GENESIS)
-					.noLogging();
+			return noOp();
 		}
 
 		return scheduleCreate("schedule-" + getHostName() + "-" +

--- a/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/CreateTokensBeforeReconnect.java
+++ b/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/CreateTokensBeforeReconnect.java
@@ -32,10 +32,10 @@ import java.util.concurrent.atomic.AtomicInteger;
 import java.util.function.Supplier;
 
 import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
-import static com.hedera.services.bdd.spec.queries.QueryVerbs.getVersionInfo;
 import static com.hedera.services.bdd.spec.transactions.TxnVerbs.tokenCreate;
 import static com.hedera.services.bdd.spec.utilops.LoadTest.defaultLoadTest;
 import static com.hedera.services.bdd.spec.utilops.UtilVerbs.logIt;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.noOp;
 import static com.hedera.services.bdd.suites.reconnect.CreateAccountsBeforeReconnect.DEFAULT_MINS_FOR_RECONNECT_TESTS;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.BUSY;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.PLATFORM_TRANSACTION_NOT_CREATED;
@@ -67,10 +67,7 @@ public class CreateTokensBeforeReconnect extends HapiApiSuite {
 	private HapiSpecOperation generateTopicCreateOperation() {
 		final long token = tokenNumber.getAndIncrement();
 		if (token >= TOKEN_CREATION_LIMIT) {
-			return getVersionInfo()
-					.fee(ONE_HUNDRED_HBARS)
-					.payingWith(GENESIS)
-					.noLogging();
+			return noOp();
 		}
 
 		return tokenCreate("token" + token)

--- a/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/CreateTopicsBeforeReconnect.java
+++ b/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/CreateTopicsBeforeReconnect.java
@@ -32,10 +32,10 @@ import java.util.concurrent.atomic.AtomicInteger;
 import java.util.function.Supplier;
 
 import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
-import static com.hedera.services.bdd.spec.queries.QueryVerbs.getVersionInfo;
 import static com.hedera.services.bdd.spec.transactions.TxnVerbs.createTopic;
 import static com.hedera.services.bdd.spec.utilops.LoadTest.defaultLoadTest;
 import static com.hedera.services.bdd.spec.utilops.UtilVerbs.logIt;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.noOp;
 import static com.hedera.services.bdd.suites.reconnect.CreateAccountsBeforeReconnect.DEFAULT_MINS_FOR_RECONNECT_TESTS;
 import static com.hedera.services.bdd.suites.reconnect.CreateAccountsBeforeReconnect.DEFAULT_THREADS_FOR_RECONNECT_TESTS;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.BUSY;
@@ -64,10 +64,7 @@ public class CreateTopicsBeforeReconnect extends HapiApiSuite {
 	private synchronized HapiSpecOperation generateTopicCreateOperation() {
 		final long topic = topicNumber.getAndIncrement();
 		if (topic >= TOPIC_CREATION_LIMIT) {
-			return getVersionInfo()
-					.fee(ONE_HUNDRED_HBARS)
-					.payingWith(GENESIS)
-					.noLogging();
+			return noOp();
 		}
 
 		return createTopic("topic" + topic)

